CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```

Comment:
So were going to say unreleased for every change we add that was not released?

Reply:
Basically that's the format - https://keepachangelog.com/en/1.0.0/

The idea is that it plays nicely with merges and stuff